rna-seq-pipeline.wdl

```diff
@@ -126,8 +123,9 @@ workflow rna {
         }
     }
 
-    # if there are exactly two replicates, calculate the madQC metrics and draw a plot
 
+
+# if there are exactly two replicates, calculate the madQC metrics and draw a plot
```

Comment:
Nit:  Rather than reiterate what the code does, perhaps this comment could explain why two replicates is a special number?